README.md

```diff
@@ -0,0 +1,54 @@
+### Before instalating you need to have installed some programs:
```

Comment:
I think someone would see only README and someone only install.sh. I think both needs this descriptions.